client/src/catalog-element-readme.html

```diff
@@ -211,6 +211,13 @@
         }
       },
 
+      _remapLinks: function(data, readme) {
+        var githubBase = 'https://github.com/' + data.owner + '/' + data.repo;
+        readme = readme.replace(/(src=")(?!http)\/?([^"]*")/g, '$1' + githubBase + '/raw/master/$2');
+        readme = readme.replace(/(href=")(?!http)\/?([^"]*")/g, '$1' + githubBase + '/blob/master/$2');
```

Comment:
@samuelli I think we are missing a way to link a page on wc.org. For example:
If I go to [app-layout on wc.org](https://beta.webcomponents.org/element/PolymerElements/app-layout) and click on app-box, I want to say on [wc.org](https://beta.webcomponents.org/element/PolymerElements/app-layout/app-box).

Meanwhile, If I'm on [Github](https://github.com/PolymerElements/app-layout), I want to stay on Github (and the on branch/commit) when I click to go to an element.